contracts/adapters/AutoRepay.sol

```diff
@@ -215,6 +228,14 @@ contract AutoRepay is BaseUniswapAdapter {
     _checkHealthFactorInRange(repayParams.user);
   }
 
+  /**
+   * @dev does swap and pull and takes 0.1% fee for caller
```

Comment:
The concepts of "swap" and "pull" are not clear. Need to describe the movement of tokens and what is swapped.